src/const.js

```diff
@@ -163,3 +163,7 @@ export const TEMPORARY_APIS = [
   'storage.local',
   'storage.sync',
 ];
+
+// Minimum sizes for icons in pixels.
+export const MIN_ICON_SIZE = 16;
+export const RECOMMENDED_ICON_SIZE = 48;
```

Comment:
I'm not sure. I don't think so but we obviously don't need a 4K image here. It might make sense to limit it to 256 just cause we can.